File: httpsnippet/src/test/resources/output/java/okhttp/application-form-encoded.java
OkHttpClient client = new OkHttpClient();

Request request = new Request.Builder()
  .url("http://mockbin.com/har")
  .post(null)
  .addHeader("content-type", "application/x-www-form-urlencoded")
  .build();

Response response = client.newCall(request).execute();

File: src/test/resources/output/javascript/fetch/application-form-encoded.js
const details ={"foo":"bar","hello":"world"};
const form = Object.entries(details)
    .map(([key, value]) => encodeURIComponent(key) + '=' + encodeURIComponent(value))
    .join('&')
const fetchOptions = {
  "mode": "cors",
  "method": "POST",
  "headers": {
    "content-type": "application/x-www-form-urlencoded"
  },
  "body": form
}

fetch("http://mockbin.com/har", fetchOptions)
 .then(response => response.json())
 .then(data => console.log(data))
 .catch(error => console.log(error));